errors/definitions.go

```diff
@@ -0,0 +1,135 @@
+package errors
+
+import (
+	"github.com/bitcoin-sv/spv-wallet/models"
+)
+
+// CONFIG ERRORS
+var (
+	// ErrDomainMissing is the error for missing domain
+	ErrDomainMissing = models.SPVError{Message: "domain is missing", StatusCode: 500, Code: "error-configuration-domain-missing"}
+
+	// ErrPortMissing is when the port is not found
+	ErrPortMissing = models.SPVError{Message: "missing a port", StatusCode: 500, Code: "error-configuration-port-missing"}
+
+	// ErrServiceNameMissing is when the service name is not found
+	ErrServiceNameMissing = models.SPVError{Message: "missing service name", StatusCode: 500, Code: "error-configuration-service-name-missing"}
+
+	// ErrCapabilitiesMissing is when the capabilities struct is nil or not set
+	ErrCapabilitiesMissing = models.SPVError{Message: "missing capabilities struct", StatusCode: 500, Code: "error-configuration-capabilities-missing"}
+
+	// ErrBsvAliasMissing is when the bsv alias version is missing
+	ErrBsvAliasMissing = models.SPVError{Message: "missing bsv alias version", StatusCode: 500, Code: "error-configuration-bsv-alias-missing"}
+
+	// ErrServiceProviderNil is the error for having a nil service provider
+	ErrServiceProviderNil = models.SPVError{Message: "service provider is nil", StatusCode: 500, Code: "error-configuration-service-provider-nil"}
+)
+
+// CAPABILITY ERRORS
+var (
+	//ErrPrefixOrDomainMissing is when the prefix or domain is missing
+	ErrPrefixOrDomainMissing = models.SPVError{Message: "prefix or domain is missing", StatusCode: 400, Code: "error-capabilities-prefix-or-domain-missing"}
+
+	//ErrDomainUnknown is when the domain is not in the list of allowed domains
+	ErrDomainUnknown = models.SPVError{Message: "prefix or domain is missing", StatusCode: 400, Code: "error-capabilities-prefix-or-domain-missing"}
```

Comment:
this error has same message and code as `ErrPrefixOrDomainMissing`

Reply:
Fixed